docs/content/references/cli/ptb.mdx

```diff
@@ -77,6 +77,11 @@ To pass in multiple types, delimit them with a comma:
 ...
 ```
 
+Structs and enums can't be directly provided as parameter, as they can only be constructed in the module in which they are declared.
+One can only call a function which returns one and then provide the result as argument to another function.
+Otherwise anyone could create any strutc with any values, for example just create new coins out of thin air.
+For enums it's limited so someone can't construct an instance that breaks some internal invariant.
```

Comment:
```suggestion
Structs and enums can't be directly provided as parameters, as they can only be constructed in the module in which they are declared.
One can only call a function that returns one and then provide the result as an argument to another function.
Otherwise anyone could create any strutc with any values, for example just create new coins out of thin air.
For enums, it's limited, so someone can't construct an instance that breaks some internal invariant.
```